tests/unit/App/Validator/Role/UpdateTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * Unit tests for Signature Verifier
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application\Tests
+ * @copyright  2013 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Tests\Unit\Core\Tool;
+
+use Kohana\Validation\Validation;
+use Ushahidi\App\Validator\Role\Update;
+use Tests\TestCase;
+use Mockery as M;
+use Ushahidi\Core\Entity\PermissionRepository;
+
+/**
+ * @backupGlobals disabled
+ * @preserveGlobalState disabled
+ */
+class RoleUpdateTest extends TestCase
+{
+
```

Comment:
Ideally we should add a test that actually calls Update::check() with some data too.